hugo/layouts/_default/list.algolia.json

```diff
@@ -0,0 +1,53 @@
+{{/* Generates a valid Algolia search index */}}
+{{- $section := $.Site.GetPage "section" .Section }}
+[
+    {{ range $idx, $page := $.Site.Pages -}}
+    {{- if $idx -}},{{- end -}}
+        {{- if or (and ($page.IsDescendant $section) (and (not $page.Draft) (not $page.Params.private))) $section.IsHome -}}
+            {{- template "algolia-object" (dict
+                "index" 0
+                "page" $page
+            ) -}}
+        {{- end -}}
+    {{- end -}}
+]
+
+{{- define "algolia-object" -}}
+    {{ $page := .page -}}
+    {{ $cutoff := 1000 -}}
+    {{ $index := .index -}}
+    {{ $skipWords := mul $cutoff $index }}
+
+    {{- $objectID := (print $page.File.UniqueID "_" (add $index 1)) -}}
+    {{- $title :=  $page.Title -}}
+    {{- $link := $page.Permalink -}}
+    {{- $categories := cond (isset $page.Params "categories") ($page.Params.categories) "" -}}
+    {{- $order := (add $index 1) -}}
+    {{- $publishDate := $page.PublishDate -}}
+    {{- $summary := $page.Summary -}}
+    {{- $content := "" -}}
+
+    {{- if eq $index 0 -}}
+        {{- $content = delimit (first $cutoff $page.PlainWords) " " -}}
+    {{- else -}}
+        {{- $content = delimit (first $cutoff (after $skipWords $page.PlainWords)) " " -}}
+    {{- end -}}
+
+    {
+        "objectID": {{ $objectID | jsonify }},
+        "order": {{ $index | jsonify }},
+        "title": {{ $title | jsonify }},
+        "categories": {{ $categories | jsonify }},
+        "link": {{ $link | jsonify }},
+        "publishDate": {{ $publishDate | jsonify }},
+        "summary": {{ $summary | jsonify }},
+        "content": {{ $content  | jsonify}}
+    }
+    
+    {{- if gt $page.PlainWords (add $cutoff $skipWords) -}},
```

Comment:
I thought you mentioned that one of our pages trips the size limit on the index?

Where is the distinct setting applied?

https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/how-to/indexing-long-documents/

Reply:
We have set something up for that as algolia told us but inside algolia it didn't look like expected. When we start using algolia search enpoint we have to see if we set it up correctly and check what results it gives us back. I'm not sure if Anne for now turned if off to be able to see if indexing worked correctly with the partial page entries